media/parent.svg

```diff
@@ -1 +0,0 @@
-<svg width="18" height="16" viewBox="0 0 18 16" xmlns="http://www.w3.org/2000/svg"><path fill="#727272" d="M12.706 6.353l-1.504 1.503-3.79-3.8v9.709H18v2.117H5.294V4.055l-3.79 3.801L0 6.353 6.353 0z" fill-rule="evenodd"/></svg>
```

Comment:
Hmm so we're getting rid of the parent button entirely? Does this mean we're relying on the arrow navigation to get there and can't jump quickly?